src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -62,6 +64,11 @@
 _AZ_LOGIN_MESSAGE = "Please run 'az login' to setup account."
 
 
+_AZURE_CLI_SUBSCRIPTION_ID = 'AZURE_CLI_SUBSCRIPTION_ID'
+_AZURE_CLI_TENANT_ID = 'AZURE_CLI_TENANT_ID'
+_AZURE_CLI_ACCESS_TOKEN = 'AZURE_CLI_ACCESS_TOKEN'
```

Comment:
I explicitly choose `AZURE_CLI_` prefix instead of `AZURE_` to avoid conflicting with other client tools, such as Azure SDK.